### PR TITLE
Preparing for first release of generated dataproc client

### PR DESCRIPTION
**Pull request by jbolinger**
First try following the python documentation.

### CODE REVIEW

File: dataproc/README.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+Python Client for Google Cloud Dataproc API (`Alpha`_)
+======================================================
+
+`Google Cloud Dataproc API`_: Manages Hadoop-based clusters and jobs on Google Cloud Platform.
```
Comment (lukesneeringer) [inline dataproc/README.rst]: Stupid question: Is this thing really called "Dataproc"?

Comment (lukesneeringer) [reply dataproc/README.rst]: [Yes!](https://cloud.google.com/dataproc/)

**Comment by jbolinger** [comment]
@lukesneeringer - want me to wait to close this out until we go through the rest of the process? Let me know when you've got time.

**Comment by lukesneeringer** [comment]
Merge away.

**Comment by dhermes** [comment]
@jbolinger @lukesneeringer This merged with broken builds.

In particular, it has broken the [docs build][1]

[1]: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/4863

**Comment by dhermes** [comment]
The spanner breakage in [your build][1] was fixed in #4620 and rebasing against `HEAD` in `master` would have allowed you to run your tests all the way through. (In fact, the **only** reason Spanner tests ran was because this branch was behind `master`.)

[1]: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/4845

**Pull request merged**